ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,6 @@
+<!-- Hello! If you're filing a bug, please include every step so as to help us reproduce it on our machines. If you're unsure about how to file an issue, use the issue template. If you need any help regarding usage of coala, check out the documentation or hit us up on chat. You can ignore or delete this text, it is commented and won't appear when the issue is submitted or previewed.
```

Comment:
> use the issue template

Isn't this the issue template? :O

Reply:
yup; that occurs at https://github.com/coala/coala/blob/master/ISSUE_TEMPLATE#L1

clearly circular and needs fixing.